crates/astria-cli/src/sequencer/sign.rs

```diff
@@ -0,0 +1,48 @@
+use astria_core::{
+    self,
+    generated::protocol::transaction::v1::TransactionBody as TransactionBodyProto,
+    protocol::transaction::v1::TransactionBody,
+};
+use color_eyre::eyre::{
+    self,
+    WrapErr as _,
+};
+
+use crate::utils::signing_key_from_private_key;
+
+#[derive(clap::Args, Debug)]
+pub(super) struct Command {
+    /// The pbjson for submission
+    pbjson: String,
```

Comment:
The semantics of this are off. This should take an input file and produce an output file.

Reply:
Replaced with a positional argument: either reads from a file or STDIN (if the trailing argument is`-`).